openmicroscopy/openmicroscopy
# Viewer script

Paris 2015 feedback (Jim at JAX)
* add option to run script from the image viewer

To test:
 * Open an image in full view
 * Check that a button to run script is visible in viewer toolbar
 * Click on it,
 * Check that the script menu pops up
